osc-server/src/main/java/org/osc/core/broker/util/crypto/X509TrustManagerFactory.java

```diff
@@ -48,29 +48,29 @@
 import org.osc.core.broker.service.ssl.CertificateResolverModel;
 import org.osc.core.broker.service.ssl.TruststoreChangedListener;
 import org.osc.core.broker.service.ssl.X509TrustManagerApi;
-import org.slf4j.LoggerFactory;
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 import org.osgi.service.component.annotations.ReferenceCardinality;
 import org.osgi.service.component.annotations.ReferencePolicy;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Component(service = X509TrustManagerApi.class, immediate=true)
 public final class X509TrustManagerFactory implements X509TrustManager, X509TrustManagerApi {
 
     private static final Logger LOG = LoggerFactory.getLogger(X509TrustManagerFactory.class);
     private static final String KEYSTORE_TYPE = "JKS";
-    // vmidctruststore stores public certificates needed to establish SSL connection
-    public static final String TRUSTSTORE_FILE = "vmidctruststore.jks";
+    // osctrustore stores public certificates needed to establish SSL connection
+    // osctrustore also stores private certificate used by application to enable
+    // HTTPS - it's also used to establish connection internally
+    public static final String TRUSTSTORE_FILE = "osctrustore.jks";
     // key entry to properties file that contains password
     private static final String TRUSTSTORE_PASSWORD_ENTRY_KEY = "truststore.password";
     // alias to truststore password entry in PKC#12 password
     private static final String TRUSTSTORE_PASSWORD_ALIAS = "TRUSTSTORE_PASSWORD";
-    // vmidckeystore stores private certificate used by application to enable HTTPS - it's also used to establish connection internally
     private static final String INTERNAL_KEYSTORE_PASSWORD_ENTRY = "internal.keystore.password";
```

Comment:
is this still needed? or maybe the comments/names need to be updated?

Reply:
The property is still used. I added back the corrected comment.